src/components/form/index.tsx

```diff
@@ -44,52 +44,62 @@ export class SmoothlyForm implements Changeable, Clearable, Submittable {
 	}
 	@Listen("smoothlyInput")
 	async smoothlyInputHandler(event: CustomEvent<Record<string, any>>): Promise<void> {
-		this.notice = undefined
-		this.smoothlyFormInput.emit(
-			(this.value = Object.entries(event.detail).reduce(
-				(r, [name, value]) => Data.set(r, name.split("."), value),
-				this.value
-			))
-		)
-		if (Clearable.is(event.target)) {
-			const clearable = event.target
-			Object.keys(event.detail).forEach(key => this.clearables.set(key, clearable))
-		}
+		this.smoothlyFormInput.emit((this.value = Data.merge(this.value, event.detail)))
 	}
 	@Listen("smoothlySubmit")
 	async smoothlySubmitHandler(event: CustomEvent): Promise<void> {
 		this.processing = true
-		this.submit()
+		await this.submit()
 		this.processing = false
 	}
 	@Listen("smoothlyInputLoad")
-	async SmoothlyInputLoadHandler(event: CustomEvent<(parent: SmoothlyForm) => void>): Promise<void> {
+	async smoothlyInputLoadHandler(event: CustomEvent): Promise<void> {
+		event.stopPropagation()
 		event.detail(this)
+		if (Input.Element.is(event.target)) {
+			this.value = Data.merge(this.value, { [event.target.name]: event.target.value })
+			this.inputs.set(event.target.name, event.target)
+		}
 	}
 	@Method()
 	async submit(): Promise<void> {
 		this.smoothlyFormSubmit.emit(this.value)
-		if (this.action) {
-			const response = await http.fetch(
-				this.method == "POST"
-					? { method: "POST", url: this.action, body: this.value }
-					: { url: `${this.action}?${http.Search.stringify(this.value)}` }
-			)
-			if (response.status >= 200 && response.status < 300) {
-				this.notice = Notice.succeeded("Form sucessfully submitted.")
-				await this.clear()
-			} else
-				this.notice = Notice.failed("Failed to submit form.")
-		}
+		this.notice?.emit(
+			Notice.execute("Submitting form", async () => {
+				let result: [boolean, string] = [false, "No action available"]
```

Comment:
Don't need to set starting values since this should always be overwritten before return.

Also, I think it's better to have the `let result` right before the if statement, since you don't need it earlier - (although this is just my preference)